test/salt/pillar/postgres.sls

```diff
@@ -1,7 +1,7 @@
 # Port to use for the cluster -- can be used to provide a non-standard port
 # NOTE: If already set in the minion config, that value takes priority
 
-{%- if not (grains.os_family == 'Debian' or grains.osfinger == 'Leap-15') %}
+{%- if not (grains.os_family == 'Debian' or ('finger' in grains and grains.osfinger == 'Leap-15')) %}
```

Comment:
I guess the string should be `osfinger` instead `finger`.
But why so complicated? We could use the same `salt['grains.get']()` function in Pillar file as well.

Reply:
Good catch - thanks.